src/Native/build-native.sh

```diff
@@ -243,18 +248,29 @@ while :; do
         arm64)
             __BuildArch=arm64
             ;;
-        debug)
+        debug|-debug)
             __BuildType=Debug
             ;;
-        release)
+        release|-release)
             __BuildType=Release
             __CMakeArgs=RELEASE
             ;;
+        -osgroup*)
+             __BuildOS=`echo $1 | cut -d '=' -f2`
+             __OSGroupSet=1
+             ;;
+        --osgroup*)
+            # msbuild adds extra - for -OSGroup
+            __BuildOS=`echo $1 | cut -d '=' -f2`
+            __OSGroupSet=1
+            ;;
         freebsd)
             __BuildOS=FreeBSD
             ;;
         linux)
-            __BuildOS=Linux
+            if [ $__OSGroupSet -eq 0 ]; then
```

Comment:
@wfurt could you please reply to this question?

Reply:
It seems like this part got lost in another section @janvorli 

when you call  "build.sh -OSGroup=FreeBSD" from top
it calls native build with run.exe as
/home/furt/git/wfurt2-corefx/src/Native/build-native.sh --OSGroup=FreeBSD x64 Debug Linux --numproc 4 -portable toolSetDir=c:\tools\clr

This may be question for build experts or fix in buildtools. OSGroup is passed in first and it it overridden by 'Linux' later. I'm assuming that the cross-patform OSGroup is not widely used so I give it precedence. I did experiment with -OS parameter  but that did not quite work either. I want "host" parts to still be Linux but "target" parts to be what over OS I choose -> like FreeBSD.

Reply:
I revert the OSGroup change. You can run build.sh -OS=FreeBSD and that avoids the problem with passing wrong parameters to build.native.sh. It would be nice if that is done properly also for OSGroup - since that works properly for managed code - but that is fix outside of corefx.